_board/solderparty_bbq20kbd.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: download
+board_id: "solderparty_bbq20kbd"
+title: "BBQ20KBD Download"
+name: "BBQ20KBD"
+manufacturer: "Solder Party"
+board_url: "https://www.solder.party/docs/bbq20kbd/"
+board_image: "solderparty_bbq20kbd.jpg"
+date_added: 2022-08-20
+family: raspberrypi
+bootloader_id: solderparty__bbq20kbd
+downloads_display: true
+blinka: false
+download_instructions: ""
+
+features:
+  - USB-C
+  - STEMMA QT/QWIIC
+---
+
+## Official description of the BBQ20KBD running official firmware

```

Comment:
Looking at other boards, don't think a header is needed between the table and the description.